coding/index.md

```diff
@@ -1,14 +1,49 @@
-# Coding convention
+# Software Code Standards
```

Comment:
this standard is only for `C` language, should we emphasize it somewhere? (it doesn't speak about eg. Assembler)